Components/Button/button.css

```diff
@@ -0,0 +1,158 @@
+* {
+    box-sizing: border-box;
+    font-family: sans-serif;
+}
+
+:root {
+    /* primary-button */
+    --primary-button-bg-color: #bae6fd;
+    --primary-button-color: black;
```

Comment:
same as above try to maintain consistency

Reply:
Yes would do that now, thanks